src/main/java/com/orange/main/products/bo/Products.java

```diff
@@ -0,0 +1,71 @@
+package com.orange.main.products.bo;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "products")
+public class Products {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String name;
+    private String description;
+    private String types;
+    private String createdDate;
+    private String updatedDate;
```

Comment:
Try to handle these with JAVA time type. Time type can be calculated and include the timezone.